parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -13,6 +13,10 @@ K8S_DOWNLOADS_DIR="/opt/kubernetes/downloads"
 UBUNTU_RELEASE=$(lsb_release -r -s)
 TELEPORTD_PLUGIN_DOWNLOAD_DIR="/opt/teleportd/downloads"
 TELEPORTD_PLUGIN_BIN_DIR="/usr/local/bin"
+KRUSTLET_DOWNLOAD_DIR="/opt/KRUSTLET/downloads"
+KRUSTLET_BIN_DIR="/usr/local/bin"
+KRUSTLET_VERSION="v0.7.0"
+KRUSTLET_URL="https://acs-mirror.azureedge.net/krustlet/${KRUSTLET_VERSION}/linux/amd64/krustlet-wasi"
```

Comment:
oops -- I duplicated this below. either way is fine.

Reply:
ironically, this is the correct prod url, and the one below is the correct staging url.